partner_gogocartojs/controllers/controllers.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from odoo import http
+
+class PartnerGogocartojs(http.Controller):
+    @http.route('/partner_gogocartojs/partner_gogocartojs/', auth='public')
```

Comment:
Why partner_gogocartojs two times ? think /partner_gogocartojs/xxx.json is better ?

Reply:
yes, it is just the hello world route